deps/verifier/test_data/cca/README.md

```diff
@@ -0,0 +1,85 @@
+Set `TRUSTEE_SRC` to the absolute path of the checked out trustee repo:
```

Comment:
We could put the README for cca directly under https://github.com/confidential-containers/trustee/tree/main/deps/verifier/src/cca like [ibmse does](https://github.com/confidential-containers/trustee/tree/main/deps/verifier/src/se)